zerver/migrations/0331_split_create_stream_policy.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import F
+
+
+def copy_stream_policy_field(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    Realm.objects.all().update(create_public_stream_policy=F("create_stream_policy"))
+    Realm.objects.all().update(create_private_stream_policy=F("create_stream_policy"))
```

Comment:
We should write a reverse migration function, that sets `create_stream_policy` to be the private stream policy.

Obviously that destroys information, but the main use case if deploying this commit goes awry and one immediately reverts, in which case it's likely their values will be identical.

(And we should have a comment in the reverse function recording this decision).